src/terms.js

```diff
@@ -103,12 +103,13 @@ export default [
     'cost control',
     'craftsmanship',
     'critical path',
+    'critics claim',
     'crypto.currency',
     'crypto(?!graphy).\\w+',
     'crm',
     'cross.sell',
     'crowd.?(fund(s?|ed|ing)|sourc(ed|e|ing))',
-    'customer obsession',
+    'customer-? ?(obsession|centric)',
```

Comment:
`customer-centric` is covered by the first entry in the list — perhaps we could just add "obsession" as the second word there?